### PR TITLE
Pawn promotion

### PR DESCRIPTION
**Pull request by djacobs74**
trying to keep it simple by updating the type of piece and the associated image
added Queen image files and merged updated master

### CODE REVIEW

**Comment by mbrown78** [comment]
we do have it. It is the unicode. Pull #22

File: app/models/pawn.rb [hunk 1]
```diff
@@ -38,5 +38,20 @@ def first_move
...
   end
-   # to be defined diagonal move when capturing other piece
+  # to be defined diagonal move when capturing other piece
+
+  # method for Pawn Promotion
+  def pawn_promotion?(x, y)
+    if y == 0 || y == 7
+      # temporarily setting choice to Queen, need to implement a choice of Bishop, Queen, Rook or Knight
+      if (:color == 'white') 
+        update_attributes(:type => Queen, :image => '&#9813')
```
Comment (lasley) [inline app/models/pawn.rb]: Updating the type should just work. The images are defined via static methods, so they should just come with type

**Comment by djacobs74** [comment]
removed image from update attributes

File: .byebug_history [hunk 2]
```diff
@@ -0,0 +1,11 @@
+continue
```
Comment (lasley) [inline .byebug_history]: This file should not be checked into source control

**Comment by saakuffo** [comment]
Conflicts need to be resolved to be merged

**Pull request closed without merging**